pkg/generate/config/field.go

```diff
@@ -136,6 +136,10 @@ type FieldConfig struct {
 	// that owns the resource. This is a special field that we direct to
 	// storage in the common `Status.ACKResourceMetadata.OwnerAccountID` field.
 	IsOwnerAccountID bool `json:"is_owner_account_id"`
+	// ReferencedType is the Group Version Kind of another CRD that can be referenced
+	// to get the value of this field. The format is <group>/<version>.<kind>.
+	// For example: "sns/v1alpha1.Topic"
+	ReferencedType *string `json:"referenced_type,omitempty"`
```

Comment:
What about using the `Type` field being introduced in this PR?

https://github.com/aws/aws-controllers-k8s/pull/636

Reply:
I am not sure but from what I can tell, `Type` is used to override the type of the field, right? `ReferencedType` is the type of the referenced object. For example, `tableNameRef` resolver should [look](https://github.com/crossplane/provider-aws/blob/f554eb9/apis/dynamodb/v1alpha1/referencers.go#L37) for `Table` resources.